app/routes/goal_routes.py

```diff
@@ -1 +1,75 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, request, Response, abort, make_response
+import requests
+import os
+from .route_utilities import validate_model, create_model
+from ..models.goal import Goal
+from ..models.task import Task
+from ..db import db
+
+
+bp=Blueprint("goal_bp", __name__, url_prefix="/goals")
```

Comment:
Nitpick: spacing is off
```suggestion
bp = Blueprint("goal_bp", __name__, url_prefix="/goals")
```